packages/quantic/force-app/main/default/lwc/quanticTooltip/quanticTooltip.js

```diff
@@ -55,8 +57,15 @@ export default class QuanticTooltip extends LightningElement {
 
   updateTooltipMaxWidth() {
     const windowWidth = window.innerWidth;
+    const targetWidth = getAbsoluteWidth(this.target);
+
+    const adaptedWidth =
+      targetWidth > 0
+        ? Math.min(windowWidth, targetWidth * targetWidthToTooltipWidthRatio)
```

Comment:
if I understand properly, we will set the width of the tooltip to 1.2 * the width of the citation if the width of the citation is > 0?